rs/sns/swap/tests/swap.rs

```diff
@@ -4027,7 +4019,7 @@ fn test_create_sns_neuron_recipes_includes_hotkeys() {
         neurons_fund_participation_icp_e8s: Some(100 * E8),
         cf_participants: vec![CfParticipant {
             controller: Some(PrincipalId::new_user_test_id(1001)),
-            hotkey_principal: PrincipalId::new_user_test_id(1001).to_string(),
+            hotkey_principal: String::new(),
```

Comment:
Nit

```suggestion
            hotkey_principal: ic_nervous_system_common::obsolete_string_filed("hotkey_principal", Some("controller")),
```

which would effect in

```suggestion
            hotkey_principal: "Field `hotkey_principal` is obsolete, please use field `controller` instead.".to_string(),
```

`ic_nervous_system_common::obsolete_string_filed("hotkey_principal", None))` could evaluate to: 

```
"Field `hotkey_principal` is obsolete.".to_string()
```

Reply:
If you decide to do this, ditto elsewhere.

Reply:
Done!